lua/orgmode/capture/templates.lua

```diff
@@ -72,4 +88,47 @@ function Templates:_compile_dates(content)
   return content
 end
 
+---@param content string
+---@return string
+function Templates:_compile_prompts(content)
+  for exp in content:gmatch('%%%^%b{}') do
+    local details = exp:match('%{(.*)%}')
+    local parts = vim.split(details, '|')
+    local title, default = parts[1], parts[2]
+    local response
+    if #parts > 2 then
+      local completion_items = vim.list_slice(parts, 3, #parts)
+      local prompt = { string.format('%s [%s]:', title, default) }
+      for i, item in ipairs(completion_items) do
+        table.insert(prompt, i .. '. ' .. item)
+      end
+      local response_number = vim.fn.inputlist(prompt)
+      response = response_number == 0 and default or completion_items[response_number + 1]
```

Comment:
You don't need `+ 1` here since prompt title is added to the `prompt` variable, not `completion_items` variable. This currently causes `1` to select option `2`.